src/common-principles.md

```diff
@@ -536,12 +547,40 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information such as physiological recordings MAY be stored with
+[compressed tab-delineated (TSVGZ) files](glossary.md#tsvgz-extensions).
+Rules for formatting plain-text tabular files apply to TSVGZ files with three exceptions:
+
+1.  The contents of TSVGZ files MUST be compressed with
+    [gzip](https://datatracker.ietf.org/doc/html/rfc1952).
+1.  Compressed tabular files MUST NOT contain a header in the first row
+    indicating the column names.
+1.  TSVGZ files MUST be accompanied by a JSON file with the same name as their
+    corresponding tabular file but with a `.json` extension.
+
+???+ warning "Columns of TSVGZ files MUST be defined in the corresponding JSON sidecar and the tabular content MUST NOT include a header line."
+
+    In contrast to plain-text TSV files, compressed tabular files files
+    MUST NOT include a header line.
+    Column names MUST be specified in the JSON file following the
+    [`Columns` metadata](glossary.md#columns-metadata) specifications
+    provided with plain-text tabular data.
+    Similarly, specific column metadata stored with dictionaries such as
+    the `sex` column in the example above MUST be specified following the
+    plain-text tabular metadata prescriptions.
+
+    TSVGZ are header-less to improve compatibility with existing software
+    (for example, FSL, or PNM), and to facilitate the support for other file formats
+    in the future.
```

Comment:
Again, too much content in the admonition title. Reading this through again, I would change this to info and write:

```suggestion

!!! note "Column definitions in compressed tabular files"

    In contrast to plain-text TSV files,
    compressed tabular files files MUST NOT include a header line.
    Column names MUST be provided in the JSON file with the
    [`Columns`](glossary.md#columns-metadata) field.
    Each column MAY additionally be described with a column description,
    as described in [Tabular files](#tabular-files).

    TSVGZ are header-less to improve compatibility with existing software
    (for example, FSL, or PNM), and to facilitate the support for other file formats
    in the future.
```